src/tests/ftest/datamover/copy_negative.yaml

```diff
@@ -0,0 +1,31 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - server-B
+timeout: 600
```

Comment:
Looking at PR-3802 build 10 results, the test took 4 mints, can we reduce the timeout to 300 secs.

Reply:
Yes, I think for copy_basics I can also reduce it.